Admin-Portal/src/pages/Auth/Login.tsx

```diff
@@ -19,6 +19,42 @@ const schema = Yup.object().shape({
     .required("Email is required"),
 });
 
+interface SuccessMessageProps {
+  success: string | null;
+  setSuccess: React.Dispatch<React.SetStateAction<string | null>>;
+}
+
+const SuccessMessage: React.FC<SuccessMessageProps> = ({
+  success,
+  setSuccess,
+}) => {
+  const handleClose = () => {
+    setSuccess(null);
+  };
+  return (
+    <Snackbar
```

Comment:
There should be an existing success message component, you can just reuse that

Reply:
Yea, I saw that one, but it seems like that only takes in a boolean value. Since I want to tell the user that they should check their email, should just modify that one to make it have the extra option to take in a string value as well?